frontend/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "outDir": "./build",
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es6",
+    "jsx": "react",
+    "moduleResolution": "node",
+    "strict": true,
+    "removeComments": true,
+    "forceConsistentCasingInFileNames": true,
+    "sourceMap": true,
+    "allowSyntheticDefaultImports": true,
+    "strictNullChecks": true
```

Comment:
just a heads up! I added this option as well as I've found it to be where most of my bugs are in the past (assuming that something will be there and then the app crashing if it's not)

Reply:
Thanks so much for doing this @gmzjuliana! 🎉 

I could be wrong, but I think with `strict: true`, all the `strict*` checks are already enabled? [[ref](https://www.typescriptlang.org/tsconfig#strict)]

Reply:
ooooh thanks for teaching me this @hamstu